backbone.js

```diff
@@ -562,12 +562,14 @@
       if (!diff) return this.hasChanged() ? _.clone(this.changed) : false;
       var old = this._changing ? this._previousAttributes : this.attributes;
       var changed = {};
+      var hasChanged;
       for (var attr in diff) {
         var val = diff[attr];
         if (_.isEqual(old[attr], val)) continue;
-        changed[attr] = val;
+        changed[attr] = val || '';
```

Comment:
if changed[a] = undefined
and i post changed to server,  will not work
how to deal with it ? 
` changed[attr] = val || null;`
better?